resources/zib/zib-FunctionalOrMentalStatus.xml

```diff
@@ -0,0 +1,168 @@
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The functional or mental status provides insight into the patient’s functional and mental limitations." />
+  <purpose value="This Observation resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) FunctionalOrMentalStatus v3.2 (2020)](https://zibs.nl/wiki/FunctionalOrMentalStatus-v3.2(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-functionalormentalstatus-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/FunctionalOrMentalStatus-v3.2(2020EN)" />
+    <name value="zib FunctionalOrMentalStatus-v3.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="FunctionalOrMentalStatus" />
+      <comment value="Please note that on a functional level, FunctionalOrMentalStatus references MedicalDevice, but in FHIR this direction is reversed. Therefore, the concept MedicalDevice (NL-CM:4.26.5) is mapped on `DeviceUseStatement.reasonReference` in profile zib-MedicalDevice instead of in this profile." />
+      <alias value="FunctioneleOfMentaleStatus" />
+      <mapping>
+        <identity value="zib-functionalormentalstatus-v3.2-2020EN" />
+        <map value="NL-CM:4.26.1" />
+        <comment value="FunctionalOrMentalStatus" />
+      </mapping>
+    </element>
+    <element id="Observation.category">
+      <path value="Observation.category" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="1" />
+      <constraint>
+        <key value="zib-FunctionalOrMentalStatus-1" />
+        <severity value="warning" />
+        <human value="Either a category for funtional status finding or mental status finding is present." />
+        <expression value="where(coding.system = 'http://snomed.info/sct' and coding.code = '118228005').exists() or where(coding.system = 'http://snomed.info/sct' and coding.code = '384821006').exists()" />

```

Comment:
'Either' in line 56 suggests that only one option is possible (i.e. a status finding cannot be both functional and mental), which makes sense. Should we then change the 'or' to 'xor' (or the logical equivalent of ' xor'), or is that too restricting?
If however it is possible for a finding to belong to both categories, maybe we should change 'Either a' to 'At least one' or something similar.

Reply:
Good suggestion! Thnx. 
I changed the operator to xor. The `.category` element s 1..*, with the intention to use one of these codes, so I would say it is not too restricting.

Reply:
This change has not yet been committed, right? (You're probably waiting on the possible other changes)